siliconcompiler/core.py

```diff
@@ -2305,7 +2287,18 @@ def hash_files(self, *keypath, algo='sha256', update=True):
         if 'file' not in self.get(*keypath, field='type'):
             self.error(f"Illegal attempt to hash non-file parameter [{keypathstr}].")
         else:
-            filelist = self.find_files(*keypath)
+            vals = self.schema._getvals(*keypath)
+            # TODO: fix this. will require changing schema structure.
+            if len(vals) > 1:
+                self.logger.warning(
```

Comment:
From a practical standpoint, not a whole lot since I don't think we really enable hashing in current use cases. But I think the tracking/provenance is still considered an important feature, so should definitely get fixed soon.